arduino/main/main.ino

```diff
@@ -113,28 +133,34 @@ void loop() {
   payload[2] = h_binary >> 8;
   payload[3] = h_binary;
 
-  //debugSerial.print("\n First byte of data sent: ");
-  //debugSerial.print(payload[0], HEX);
-  //debugSerial.print("\n Second byte of data sent: ");
-  //debugSerial.print(payload[1], HEX);
-  //debugSerial.print("\n Third byte of data sent: ");
-  //debugSerial.print(payload[2], HEX);
-  //debugSerial.print("\n Fourth byte of data sent: ");
-  //debugSerial.print(payload[3], HEX);
-  //debugSerial.println();
-
-  ttn.sendBytes(payload, sizeof(payload));
-  
+  payload[4] = subtypes[0]; // indoor
+
+  #ifdef DEBUG
+  debugSerial.print("\n First byte of data sent: ");
+  debugSerial.print(payload[0], HEX);
+  debugSerial.print("\n Second byte of data sent: ");
+  debugSerial.print(payload[1], HEX);
+  debugSerial.print("\n Third byte of data sent: ");
+  debugSerial.print(payload[2], HEX);
+  debugSerial.print("\n Fourth byte of data sent: ");
+  debugSerial.print(payload[3], HEX);
+  debugSerial.println();
+  #endif
+
+  ttn_response_t response = ttn.sendBytes(payload, sizeof(payload));
+
   // Compute heat index in Fahrenheit (the default)
   // float hif = dht.computeHeatIndex(f, h);
   // Compute heat index in Celsius (isFahreheit = false)
   // float hic = dht.computeHeatIndex(t, h, false);
 
-  //debugSerial.print(F("Humidity: "));
-  //debugSerial.print(h);
-  //debugSerial.print(F("%  Temperature: "));
-  //debugSerial.print(t);
-  //debugSerial.print(F("°C "));
+  #ifdef DEBUG
+  debugSerial.print(F("Humidity: "));
+  debugSerial.print(h);
+  debugSerial.print(F("%  Temperature: "));
+  debugSerial.print(t);
+  debugSerial.print(F("°C "));
+  #endif
   //debugSerial.print(f);
   //debugSerial.print(F("°F  Heat index: "));
   //debugSerial.print(hic);
```

Comment:
Delete code that's commented out (if not useful)

Reply:
I'll do that